workflows/src/main/kotlin/com/r3/corda/lib/tokens/workflows/flows/issue/IssueTokensUtilities.kt

```diff
@@ -45,4 +46,28 @@ fun addIssueTokens(transactionBuilder: TransactionBuilder, vararg outputs: Abstr
 @Suspendable
 fun addIssueTokens(transactionBuilder: TransactionBuilder, output: AbstractToken<*>): TransactionBuilder {
     return addIssueTokens(transactionBuilder, listOf(output))
+}
+
+// Utilities for ensuring that the correct JAR which implements TokenType is added to the transaction.
+
+fun addTokenTypeJar(tokens: List<AbstractToken<*>>, transactionBuilder: TransactionBuilder) {
```

Comment:
Should we move it to internal package?

Reply:
Others that create their own flows might need to use this